src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,12 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo
+        post={post}
```

Comment:
It is recommended to destructure the post object to pass the individual props to PostInfo component. This would make the code more readable and maintainable.